pages/services/edge-lb/0.1/installing/index.md

```diff
@@ -20,9 +20,21 @@ Configure a service account and install the Edge-LB package using the instructio
 - Currently, Edge-LB works only with DC/OS Enterprise in permissive [security mode](/latest/security/ent/#security-modes). It does not work with strict or disabled.
 
 # Add Edge-LB package repositories
-The Edge-LB package is composed of two components: the Edge-LB API server and the Edge-LB pools. You must install universe repos for the Edge-LB API server and the Edge-LB pool in order to install Edge-LB. The Edge-LB API server is a restful API that manages one or more Edge-LB pools. Each Edge-LB pool is a collection of load balancers. An Edge-LB pool can be used to launch one or more instances of a load balancer to create a single highly available load balancer. Currently the Edge-LB pool supports only HAProxy as a load balancer.
+The Edge-LB package comprises two components: 
 
-Download the artifacts for each of the repos from the [Mesosphere support page](https://support.mesosphere.com/hc/en-us/articles/213198586).
+- The **Edge-LB API server** is a restful API that manages one or more Edge-LB pools. Each Edge-LB pool is a collection of load balancers. 
+
+- An **Edge-LB pool** can be used to launch one or more instances of a load balancer to create a single highly available load balancer. Currently the Edge-LB pool supports only HAProxy as a load balancer. 
+
+You must install universe repos for the Edge-LB API server and the Edge-LB pool in order to install Edge-LB. 
+
+## Obtain artifacts
+
+If your cluster already has connectivity, you can obtain the artifacts directly. 
+
+If you do not have a cluster with connectivity, you will then need a service account to download the artifacts for each of the repos from the [Mesosphere support page](https://support.mesosphere.com/hc/en-us/articles/213198586). Note that you will get a "page not found" message if you attempt to download the artifacts without a service account.
```

Comment:
`service account` -> `DC/OS support portal account`?

See also my remarks in [1].

[1] https://jira.mesosphere.com/browse/DCOS-40392?focusedCommentId=182947&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-182947